docs/hacking/compiling.rst

```diff
@@ -177,7 +177,7 @@ one or more of the other graphical front ends are selected. The graphical front
 ends are: GCU, SDL, SDL2 and X11.  All of the following generate a
 self-contained directory, build, that you can move elsewhere or rename.  To
```

Comment:
Since the executable is now in build/game.  This should be "self-contained directory, build/game, that you can move elsewhere or rename.  To run the result, change directories to build/game or whatever you renamed it to) and run ./angband ."

Reply:
Updated.
I have also added Windows as a graphical front end here.